### PR TITLE
Ember-CLI Notes 22.06.2017

### PR DESCRIPTION
**Pull request by hjdivad**

### CODE REVIEW

File: ember-cli/2017-06/june-22.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# ember-cli Meeting Notes - 2017-06-22
+
+## Attendees
```
Comment (hjdivad) [inline ember-cli/2017-06/june-22.md]: Comment your agenda items here @ember-cli/core

**Comment by twokul** [comment]
Would love to talk about introducing types to ember cli, either to [Typescript](https://www.typescriptlang.org/docs/handbook/migrating-from-javascript.html) or [Flow](https://flow.org/)

**Comment by SergeAstapov** [comment]
@twokul Isn't it a scope of projects like [ember-cli-typescript](https://github.com/emberwatch/ember-cli-typescript)?

**Comment by chriskrycho** [comment]
@SergeAstapov @twokul yeah, medium-term goal is to get ember-cli-typescript in a state where Ember and Ember CLI themselves can use it.

**Comment by twokul** [comment]
@SergeAstapov @chriskrycho maybe? I suspect requiring a build step for Ember CLI itself is abit much. My thinking is to first start introducing types for things like `options` objects in Ember CLI (we have a lot of them and it's east to get lost). The rest of the system I'm not 100% sure about just yet.

**Comment by eccegordo** [comment]
@twokul yes I think the typescript idea is interesting if it can be achieved in such a way that is transparent to most users from a tooling/compilation perspective. But if it lends itself to greater discoverability of the internal types within CLI that would be a win. Especially for stuff like addons and blueprints API surface.

**Comment by twokul** [comment]
@eccegordo I think I agree. But I'm talking about trying it out for a particular smaller use case within Ember CLI itself (just for Ember CLI contributors if you will), rather than tackling bigger use cases (like typing of bluerprints or add-ons).

**Comment by Turbo87** [comment]
> I suspect requiring a build step for Ember CLI itself is abit much

how else would this work? Node itself doesn't understand TypeScript or Flow syntax.

**Comment by kratiahuja** [comment]
ts-node does understand typescript ;)

File: ember-cli/2017-06/june-22.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+
+## Agenda
```
Comment (kratiahuja) [inline ember-cli/2017-06/june-22.md]: Would like to talk about:

1. https://github.com/ember-cli/rfcs/pull/108
2. Setting a date for FastBoot 1.0.

**Comment by eriktrom** [comment]
code splitting - IMHO is more important for time to interactive than any other feature ember could provide ATM

**Comment by Turbo87** [comment]
@eriktrom `ember-engines`?

**Comment by eriktrom** [comment]
> @eriktrom ember-engines

Thanks @Turbo87 for the response. I wasn't sure if that was *the full answer*

Is anyone working on adding ember to this list? https://hnpwa.com/ - if not i'll do it - before we launch our product, a lighthouse score in the 90's is very important.

**Comment by trentmwillis** [comment]
@eriktrom there is: https://github.com/ivanvanderbyl/ember-hackernews-pwa, which you can view at [hackernews.io](https://hackernews.io)

**Comment by chadhietala** [comment]
@eriktrom https://github.com/chadhietala/glimmer-hn The app is "done" but needs to be hosted somewhere.

File: ember-cli/2017-06/june-22.md (hunk 2 above)
Comment (stefanpenner) [inline ember-cli/2017-06/june-22.md]: using [prettier](https://github.com/prettier/prettier) on the ember-cli codebase.

**Comment by hjdivad** [comment]
cc
@kellyselden 
@kratiahuja 
@rwjblue 
@trabus 
@turbo87

**Pull request merged**